pkg/cmd/api/api.go

```diff
@@ -540,36 +540,58 @@ func parseErrorResponse(r io.Reader, statusCode int) (io.Reader, string, error)
 
 	var parsedBody struct {
 		Message string
-		Errors  []json.RawMessage
+		Errors  json.RawMessage
 	}
 	err = json.Unmarshal(b, &parsedBody)
 	if err != nil {
-		return r, "", err
+		return bodyCopy, "", err
+	}
+
+	if len(parsedBody.Errors) > 0 && parsedBody.Errors[0] == '"' {
+		var stringError string
+		if err := json.Unmarshal(parsedBody.Errors, &stringError); err != nil {
+			return bodyCopy, "", err
+		}
+		if stringError != "" {
+			if parsedBody.Message != "" {
+				return bodyCopy, fmt.Sprintf("%s (%s)", stringError, parsedBody.Message), nil
```

Comment:
Would it make sense to reverse `stringError` and `parsedBody.Message` here? That seems like it would better mimic the response sent back on line 564.

Reply:
So, `Conflict (Actions are enabled for all repositories)`?

Reply:
Yeah that was my thought. I don't have strong feelings though, it was mostly for consistency where the `parsedBody.Message` would always come first with the details following enclosed in parenthesis.

Reply:
I get that re: consistency, but in this (strange) error case it looks like the `message` fields holds the "type" of the error ("Conflict") while the `error` field is the real human-readable error (i.e. what `message` should have been). Generally, we print the human-readable message first, optionally followed by the error type in parentheses.